.ci/ci.yml

```diff
@@ -227,6 +275,8 @@ jobs:
                 TERRAFORM_REPO: terraform-providers/terraform-provider-google
                 PUPPET_COMPUTE_REPO: GoogleCloudPlatform/puppet-google-compute
                 PUPPET_SQL_REPO: GoogleCloudPlatform/puppet-google-sql
+                CHEF_COMPUTE_REPO: GoogleCloudPlatform/chef-google-compute
```

Comment:
Could this be parameterized? Something like: `GoogleCloudPlatform/chef-google-{{product}}`